plugins/modules/ome_alert_policies.py

```diff
@@ -0,0 +1,962 @@
+                          },
+        "severity": {'type': 'list', 'elements': 'str', 'choices': ['info', 'normal', 'warning', 'critical', 'unknown', 'all']},
+        "actions": {'type': 'list', 'elements': 'dict',
+                    'options': {'action_name': {'type': 'str', 'required': True},
+                                'parameters': {'type': 'list', 'elements': 'dict', 'default': [],
+                                               'options': {'name': {'type': 'str'},
+                                                           'value': {'type': 'str'}}
+                                               }
+                                }
+                    }
+    }
+    specs.update(ome_auth_params)
+    module = AnsibleModule(
+        argument_spec=specs,
+        required_if=[['state', 'present', ('enable', 'new_name', 'description', 'device_service_tag', 'device_group',
+                                           'specific_undiscovered_devices', 'any_undiscovered_devices', 'all_devices',
+                                           'category', 'message_ids', 'message_file',
+                                           'date_and_time', 'severity', 'actions',), True]],
+        mutually_exclusive=[('device_service_tag', 'device_group', 'any_undiscovered_devices', 'specific_undiscovered_devices'),
+                            ('message_ids', 'message_file', 'category')],
+        supports_check_mode=True)
+    try:
+        with RestOME(module.params, req_session=True) as rest_obj:
+            state = module.params.get('state')
+            name_list = module.params.get('name')
+            policies = get_alert_policies(rest_obj, name_list)
+            if state == 'absent':
+                if policies:
+                    remove_policy(module, rest_obj, policies)
+                else:
+                    module.exit_json(msg=NO_CHANGES_MSG)
+            else:
+                if not any(module.params.get(prm)
+                           for prm in ('new_name', 'description', 'device_service_tag', 'device_group',
+                                       'specific_undiscovered_devices', 'any_undiscovered_devices', 'all_devices',
+                                       'category', 'message_ids', 'message_file',
+                                       'date_and_time', 'severity', 'actions')) and module.params.get('enable') is not None:
+                    if len(policies) == len(set(name_list)):
+                        enable_toggle_policy(module, rest_obj, policies)
+                    else:
+                        invalid_policies = set(name_list) - set(x.get("Name") for x in policies)
+                        module.exit_json(failed=True, msg=f"Policies {SEPARATOR.join(invalid_policies)} are invalid for enable.")
+                if len(name_list) > 1:
+                    module.exit_json(failed=True, msg="More than one policy name provided for update.")
```

Comment:
Keep this error message on top

Reply:
the enable feature flow has to be checked first hence this is after enable check